lib/vmpooler/api/v1.rb

```diff
@@ -45,35 +45,39 @@ def alias_deref(hash)
       newhash
     end
 
-    def checkout_vm(template, result)
-      vm = backend.spop('vmpooler__ready__' + template)
+    def fetch_single_vm(template)
+      backend.spop('vmpooler__ready__' + template)
```

Comment:
It looks like vmpooler explicitly emits log messages:

```ruby
        $logger.log('s', "[>] [#{pool}] '#{vm}' moved to 'ready' queue")
```

So I'm thinking there shouldn't be any added verbosity. Unless our redis library also logs somewhere?